deploy/docs/NewDemoDeployment.md

```diff
@@ -0,0 +1,109 @@
+# Creating a Demo Deployment:
+
+This is very similar to creating a new deployment (NewDeployment.md), particularly in the browser/api section.
+
+However, there is additional cleanup that should be performed as demo instances are typically temporary, and should be removed after their purpose is served.
+
+There exist several Pythong scripts in the `deployctl` package that make this process straightforward. These files are located in `gnomad-browser/deploy/deployctl/subcommands`.
```

Comment:
One last typo here, "Pythong", otherwise good. What I like to do in a case like this, rather than do another full formal round of review, is to give a contingent approval, the idea being that while the typo should be fixed it's inefficient for both of us to do another round for such a small change.

So, I'm going to approve this PR now, but please fix the typo before you actually merge, is what that comes down to in this case.